src/frontend/apps/desk/src/features/teams/api/data/teams.ts

```diff
@@ -0,0 +1,26 @@
+import { faker } from '@faker-js/faker';
+
+import { Role } from '../types';
+
+export const dummyDataAPITeamAccesses = (total: number, pageSize: number) => {
+  const accesses = [];
+  const count = total;
+  const roles = Object.values(Role);
+  for (let i = 0; i < pageSize; i++) {
+    accesses.push({
+      id: faker.string.uuid(),
+      user: {
+        email: faker.internet.email(),
+        name: faker.person.fullName(),
+      },
+      role: roles[Math.floor(Math.random() * roles.length)],
```

Comment:
nit: take a look at [here](https://fakerjs.dev/api/helpers#enumvalue), might be useful for next mocks